### PR TITLE
Bump version of wasm-feature-detect

### PR DESCRIPTION
**Pull request by sbc100**
I'm hoping to get extended-const support but there are other new detectors too I think.

### CODE REVIEW

File: roadmap.js [hunk 1]
```diff
@@ -298,7 +298,7 @@
   function _loadTooltipModule() {
     // Be sure to change the preloads in markdown when updating url.
     // The ESM bundle of this package doesn't work with unpkg.com.
-    const module = import('https://cdn.jsdelivr.net/npm/@floating-ui/dom@1/+esm');
```
Comment (dschuff) [inline roadmap.js:301]: does the "1"  here mean "1.0" (i.e. pinned to an older revision) or does it mean "get me the latest 1.x version"?

**Review by dschuff — APPROVED** [review]

File: roadmap.js (hunk 1 above)
Comment (sbc100) [reply roadmap.js:301]: No idea.. I just verified that the `1` didn't give me the new version that I wanted and `1.3` did.

Comment (sbc100) [reply roadmap.js:301]: @RReverser is this the correctly thing to do here?

Comment (RReverser) [reply roadmap.js:301]: > does the "1" here mean "1.0" (i.e. pinned to an older revision) or does it mean "get me the latest 1.x version"?

I honestly don't know; before https://github.com/WebAssembly/website/pull/283 we used unpkg.com where it wouldn't be pinned, but I'm not very familiar with what jsdelivr does.

Explicitly bumping to 1.3 sounds good.

Comment (RReverser) [reply roadmap.js:301]: That said, https://cdn.jsdelivr.net/npm/@floating-ui/dom@1.3/+esm doesn't seem to work?

Comment (RReverser) [reply roadmap.js:301]: Wait, I don't think this is a module you meant to bump at all. This is not wasm-feature-detect, it's just a tooltip library.

**Comment by RReverser** [comment]
Bumped the correct library in https://github.com/WebAssembly/website/commit/2d5bcb9ef1bc8548436153e595b6385ba36d5f76 instead.

File: roadmap.js (hunk 1 above)
Comment (sbc100) [reply roadmap.js:301]: Ooops! Wrong URL.

Actually I noticed that the existing URL seems to fetch 1.3 already: https://cdn.jsdelivr.net/npm/wasm-feature-detect@1/dist/esm/index.js

So I'm closing this..

**Comment by sbc100** [comment]
Ooop, sorry for the noise.  It seem that maybe this wan't needed?

**Comment by sbc100** [comment]
Anyway looks like the new detector for extended const is online and working as intended!

**Comment by RReverser** [comment]
I think it still was - it's a bit scary because `@1` seems to resolve to whatever's latest `1.x.y` is; that is, it seems likely that between 1.3 release of wasm-feature-detect and your PR fixing the feature name https://github.com/WebAssembly/website/pull/286 we were briefly broken due to incompatible feature names.

Better to be semver-explicit.

**Comment by RReverser** [comment]
> Anyway looks like the new detector for extended const is online and working as intended!

Are there browsers where it works for you? If it's available under a flag in some browsers already, we should reflect that in features.json.

**Comment by RReverser** [comment]
Looks like we did have it before: https://github.com/WebAssembly/website/blob/master/features.json#L97

Here and in other browsers the feature needs to be renamed as well now.

**Comment by sbc100** [comment]
Yes, it works in chrome with flag, I added that in #285, but it looks like I need another update because I called it `extendedConstantExprs` instead of `extendedConst`.   Sending another PR now.

File: roadmap.js (hunk 1 above)
Comment (jimaek) [reply roadmap.js:301]: Just to clarify @1 will load the latest 1.x.x version https://github.com/jsdelivr/jsdelivr#npm
The HTTP headers also verify it:

```
x-jsd-version: 1.3.0
x-jsd-version-type: version
```

**Pull request closed without merging**